### PR TITLE
Create python/fluid/README.md 

### PR DESCRIPTION
**Pull request by wangkuiyi**

### CODE REVIEW

File: python/fluid/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+```protobuf
+message Type {
+  enum FirstClass {
+    INT32 = 1;
+    FLOAT32 = 2;
+    ...
+  }
+  message Tensor {
+    required Type elem = 1; <-- (1)
```
Comment (emailweixu) [inline python/fluid/README.md:30]: "Type" cannot be used here. It has not been completely defined yet.

**Review by emailweixu — COMMENTED** [review]
I don't understand this doc.

File: python/fluid/README.md (hunk 1 above)
Comment (wangkuiyi) [reply python/fluid/README.md:30]: I tried and it seems that protobuf syntax allows Type being used here.

**Review by wangkuiyi — COMMENTED** [review]
I understand that it is not easy to pose the problems without code. I am closing the issue and try to file another PR that includes the code soon.

**Pull request closed without merging**